### DISCUSSION TITLE
"TransitionGroup" error seems like it hasn't been fixed

### DISCUSSION THREAD

**Discussion by KingGamingYT** [tsukasa/BdBrowser#38] [Problems and issues]
![image](https://github.com/tsukasa/BdBrowser/assets/45918062/9756d31d-b592-46f5-af3f-34438c60e9a6)
When trying to use the newest version of the extension, the console throws this error, which suggests the [bugfix](https://github.com/BetterDiscord/BetterDiscord/commit/07a97e24344b6506a90efb343519d7f49170cf06) needed to get BetterDiscord working on desktop clients again either hasn't been fixed here or there's some other issue.

**Comment by tsukasa** [comment]
Greetings,

the error shown in your screenshot is more indicative of BetterDiscord not being updated and therefore failing to start.

Please try these steps:
- Open the extension options through the toolbar icon or the Extensions page.
- Activate the "Delete Asar from VFS on Reload" checkbox.
- Reload the Discord tab. A new copy of the BetterDiscord ASAR file should be downloaded into the VFS.

If you use a local copy of the renderer (via the "bd" subdirectory of the extension), make sure the files in the "bd" subdirectory are version 10.0.2 or newer.

**Comment by KingGamingYT** [comment]
Hi!

I did everything you mentioned yesterday...before the fix came out. Silly me for not thinking to retrace my steps today. Thanks!
